Megaparsec/MonadParsec.lean

```diff
@@ -328,6 +328,8 @@ instance theInstance {m : Type u → Type v} {α β ℘ E : Type u} [Streamable
   updateParserState φ := fun _ s _ _ eok _ =>
     eok.2 PUnit.unit (φ s) []
 
+open YatimaStdLib (Monoid One)
```

Comment:
What does this do? Just curious :slightly_smiling_face:

Reply:
It makes `open` more restrictive regarding what to make available from a namespace. In this case, I'm telling Lean to only pick `Monoid` and `One` from that namespace

Reply:
Yeah, no, of course, I was actually wondering why we need `One` at all. But apparently we do use it! Wonders never cease :sweat_smile: